README.md

```diff
@@ -1,23 +1,46 @@
 # vscode-kedro
-It's in a beta status so it's not uncommon to have breaking feature and bugs, report this in Github or Slack, we will try to fix ASAP.
+The extension is in beta, so you may experience breaking changes and bugs. If you encounter issue, report it in [Github](https://github.com/kedro-org/vscode-kedro/issues) or [Slack](https://slack.kedro.org), we will try to fix ASAP.
 
-## How to use this extension
-1. Install `Kedro` from the extension
-2. Select the correct Python interpreter that you use to run the Kedro project`Python: select interpreter`
-
-p.s. If you can `kedro run` with the environment it's good to go.
 
 ## Requirements
 1. VS Code 1.64.0 or greater
-1. Python extension for VS Code
+2. Python extension for VS Code
+3. Kedro Project >= 0.19
+
+## How to use this extension
+1. Install `Kedro` from the extension
+2. Select the correct Python interpreter that you use to run the Kedro project`Python: select interpreter`
 
+p.s. If you can `kedro run` with the environment, you are good to go.
 
 The extension requires `bootstrap_project` in Kedro, you need to make sure you can do `kedro run` without getting any immediate error, otherwise you may get a server panic error.
 
 ## How to restart a server if there are error
 Click `Output` and select `Kedro` from the dropdown list. It may gives you some hints and report back if you think this is a bug.
 
-Hit `Cmd` + `Shift` + `P` to open the VSCode command, look for `kedro: restart server` in case it's panic
+Hit `Cmd` + `Shift` + `P` to open the VSCode command, look for `kedro: restart server` in case it's panic.
+
+## Assumptions
+### Configuration Source
+Currently, the extension assume the source of configuration is `conf/base`. If you have customised `CONF_SOURCE` in project settings, certain features will not be functional properly.
+
+### Configuration Environment
+The extension assumes `base` environment. However, you may want it to use different environments. This is still WIP.
```

Comment:
What does this mean? That a `base` environment exists? Or that it's the `base_env`?

Reply:
https://github.com/kedro-org/vscode-kedro/issues/14#issuecomment-2173943851

It's "conf/base" at the moment, I left a comment and will address this in the ticket when we extend support for environment.

Reply:
I correct myself, I actually already implement in a more generic way. i.e. `Path(config_loader.conf_source) / config_loader.base_env)`

Reply:
I am gonna make some change in https://github.com/kedro-org/vscode-kedro/issues/14 so that it actually takes `KEDRO_ENV` if it is set.